orchagent/neighorch.cpp

```diff
@@ -551,7 +551,8 @@ void NeighOrch::doTask(Consumer &consumer)
 
         string alias = key.substr(0, found);
 
-        if (alias == "eth0" || alias == "lo" || alias == "docker0")
+        if (alias == "eth0" || alias == "lo" || alias == "docker0"
+            || ((op == SET_COMMAND) && m_intfsOrch->isInbandIntfInMgmtVrf(alias)))
```

Comment:
How about any routes pointing to inband mgmt vrf?

Reply:
Those routes would be ignored in fpmsyncd.